psl-core/src/main/java/org/linqs/psl/application/inference/online/messages/actions/controls/WriteInferredPredicates.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * This file is part of the PSL software.
+ * Copyright 2011-2015 University of Maryland
+ * Copyright 2013-2021 The Regents of the University of California
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.linqs.psl.application.inference.online.messages.actions.controls;
+
+import org.linqs.psl.application.inference.online.messages.actions.OnlineAction;
+
+/**
+ * Write inferred predicates at a specified location on the server.
+ * String format: WriteInferredPredicates <path>
```

Comment:
Specify the behavior when no path is provided.
Also, use square brackets in the format to indicate an optional parameter.